installation/install-ambari/configure-repository.rst

```diff
@@ -0,0 +1,64 @@
+Configure the Ambari Repository
+===============================
+
+Follow the instructions in the section for the operating system that runs your installation host.
+
+RHEL/CentOS
+_____________
+
+On a server host that has Internet access, use a command line editor to perform the following steps:
+
+1. Log in too your host as ``root``
+
+2. Add the |fdd| Yum repository to a file called ``/etc/yum.repos.d/ambari.repo``
+
+  **RHEL/CentOS 6.x**
+
+  ::
+
+    [Updates-ambari-2.2.0.0]
+    name=ambari-2.2.0.0 - Updates
+    baseurl=https://rbp_email_address:password@public-repo.radicalbit.io/ambari/centos6/2.2.0.0
+    gpgcheck=1
+    gpgkey=https://public-repo.radicalbit.io/repo/RPM-GPG-KEY/RPM-GPG-KEY-Radicalbit.key
+    enabled=1
+    priority=1
+
+  **RHEL/CentOS 7.x**
+
+  ::
+
+    [Updates-ambari-2.2.0.0]
+    name=ambari-2.2.0.0 - Updates
+    baseurl=https://rbp_email_address:password@public-repo.radicalbit.io/ambari/centos7/2.2.0.0
+    gpgcheck=1
+    gpgkey=https://public-repo.radicalbit.io/repo/RPM-GPG-KEY/RPM-GPG-KEY-Radicalbit.key
+    enabled=1
+    priority=1
+
+  where *rbp_email_address* and *password* are the account credentials you created on the registration page [TODO -> ADD LINK].
```

Comment:
I'm not sure it's actually online and properly working.
maybe we could open another pr when it's ok